tests/integration_tests/conftest.py

```diff
@@ -71,6 +71,8 @@ def pytest_runtest_setup(item):
     supported_os_set = set(os_list).intersection(test_marks)
     if current_os and supported_os_set and current_os not in supported_os_set:
         pytest.skip("Cannot run on OS {}".format(current_os))
+    if 'unstable' in test_marks:
```

Comment:
Is it worth adding `RUN_UNSTABLE` as a setting, so the interface for forcing a run is `CLOUD_INIT_RUN_UNSTABLE=true` rather than a code modification?